recipe/0219-using-caption-file/index.md

```diff
@@ -27,8 +27,7 @@ Formats other than [WebVTT](https://w3c.github.io/webvtt/) (Web Video Text Track
 
 In this example we use a caption file in the WebVTT format, but one could just as well use a subtitle file in the [SRT](https://en.wikipedia.org/wiki/SubRip) (SubRip Text) or [TTML](https://w3c.github.io/ttml3/index.html) (Timed Text Markup Language) formats, or other text-based format used for the same purpose.
 
```

Comment:
Could you add the link back to the manifest so it looks like this:

{% include manifest_links.html viewers="" manifest="manifest.json" %}